server/src/main/java/org/apache/iotdb/db/metadata/template/TemplateManager.java

```diff
@@ -175,9 +176,10 @@ public void checkTemplateCompatible(Template template, IMNode node) throws Metad
       for (IMNode dNode : template.getDirectNodes()) {
         if (dNode.isMeasurement()) {
           throw new MetadataException(
-              "Template and mounted node has different alignment: "
```

Comment:
Hey, this PR causes the unit test  `TemplateTest.testTemplateAlignment` failed. Could you do a patch to fix the test as well?

![image](https://user-images.githubusercontent.com/3821212/154889863-0c20d2e5-421f-4d4c-9a41-bbdf99b4578f.png)

Reply:
Sorry for that failure, I am putting forward another PR now
links:https://github.com/apache/iotdb/pull/5096